README.md

```diff
@@ -37,14 +37,19 @@ var cjsTree = typescript(inputTree, {
 });
 ```
 
-The broccoli plugin class
+### Config Options:
+
+* `tsconfig:` See: https://www.typescriptlang.org/docs/handbook/tsconfig-json.html
+
+### Ways to use:
+
+via the broccoli plugin subclass
 
 ```js
 var TypeScript = require('broccoli-typescript-compiler').TypeScript;
 ```
 
-Legacy filter behavior (compile all .ts files and only .ts and
```

Comment:
why did you remove this?

typescript is more akin to rollup than a general filter unless you specify isolated-modules in the config, it has entry-points and outputs only files compiled as part of the program via imports or root names.

I only added the filter version for backwards compatibility.